core/advancedpolls.php

```diff
@@ -346,6 +352,23 @@ public function do_poll_modification($topic_data)
 	}
 
 	/**
+	 * Add the possible options to the template
+	 *
+	 * @param array	$post_data		The array of post data
+	 * @return void
+	 */
```

Comment:
Please update this docblock with a short explaining sentence what this function does/what it is for and the correct parameter name, data type and description.

That helps me to understand the code months and years later :P And my IDE also shows the text on IntelliSense.

Ah, and while you are at it. Even an internal function like the `order_by_votes()` below should have a docblock. parameters of course, and maybe a sentence like "Function used for sorting blah bblahb blah"